### PR TITLE
Add boot order

### PR DESCRIPTION
**Pull request by dc-bradwadsworth**
This will allow a user to specify the boot order of devices.

### CODE REVIEW

File: lib/vagrant-libvirt/templates/domain.xml.erb [hunk 1]
```diff
@@ -27,7 +27,14 @@
...
     <% end %>
-    <boot dev='hd'/>
+    <% if @boot_order.count >= 1 %>
+      <% @boot_order.each do |b| %>
+    <boot dev='<%= b %>'/>
+      <% end %> 
+    <bootmenu enable='yes'/>
+    <% else %>
+    <boot dev='hd' />
+    <% end %>
```
Comment (pronix) [inline lib/vagrant-libvirt/templates/domain.xml.erb]: fix please indent

**Comment by pronix** [comment]
provide please example how to set boot order.

**Comment by dc-bradwadsworth** [comment]
Just out of curiosity, when would the change be implemented into the version that lives in Rubygems so that a `vagrant plugin install vagrant-libvirt` will get the newest version?

**Comment by pronix** [comment]
released 0.0.30

**Pull request merged**